pkg/apis/options/providers.go

```diff
@@ -132,6 +132,8 @@ type OIDCOptions struct {
 	// InsecureSkipIssuerVerification skips verification of ID token issuers. When false, ID Token Issuers must match the OIDC discovery URL
 	// default set to 'false'
 	InsecureSkipIssuerVerification bool `json:"insecureSkipIssuerVerification,omitempty"`
+	// InsecureSkipNonce skips verifying the ID Token's nonce claim
+	InsecureSkipNonce bool `json:"insecureSkipNonce,omitempty"`
```

Comment:
Perhaps we want to expand on the documentation on this field. This is what gets generated into the reference docs after all.

I'd say let's include the default as others are and maybe even the little warning about it changing in a future release